src/Microsoft.Identity.Web/InstanceDiscovery/Metadata.cs

```diff
@@ -15,19 +15,19 @@ internal class Metadata
         /// Preferred alias.
         /// </summary>
         [JsonProperty(PropertyName = "preferred_network")]
-        public string PreferredNetwork { get; set; }
+        public string? PreferredNetwork { get; set; }
```

Comment:
This case is similar to [Late-initialized properties, Data Transfer Objects and nullability](https://docs.microsoft.com/en-us/dotnet/csharp/nullable-migration-strategies#late-initialized-properties-data-transfer-objects-and-nullability). What is our intent - value is required for all these properties or optional?

Reply:
Very good point, I missed this one!

Reply:
Good thing about this work and PR is that it starts the discussions about the intent of our code. 👍